src/main/java/mate/academy/Event.java

```diff
@@ -1,4 +1,7 @@
 package mate.academy;
 
-public record Event(String type, Object source) {
+public record Event(
+        String type,
+        Object source
```

Comment:
It is a good practice to use a specific type instead of `Object` for the `source` field if possible. This can provide type safety and make the codebase easier to maintain.

Reply:
It is not